include/dpp/appcommand.h

```diff
@@ -810,11 +810,11 @@ class DPP_EXPORT slashcommand : public managed {
 	/**
 	 * @brief Construct a new slashcommand object
 	 * 
-	 * @param _name Command name
-	 * @param _description Command description
-	 * @param _application_id Application id (usually the bot's user id)
+	 * @param name Command name
```

Comment:
you can't change these, there's a reason for this (to silence doxygen warnings, and that you can't have a parameter name that shadows a name of a member variable